packaging/scripts/centreon-agent-daemon-preinstall.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if ! id centreon-agent > /dev/null 2>&1; then
```

Comment:
same comment as above regarding user group and permissions, this use is already owned by another piece of software of centreon